esphomeyaml/components/time.rst

```diff
@@ -59,6 +59,32 @@ Configuration variables:
 - **on_time** (*Optional*, :ref:`Automation <automation>`): Automation to run at specific intervals using
   a cron-like syntax. See :ref:`time-on_time`.
 
+DS1307 Time Source
+------------------
+
+With the ``DS1307`` time platform, the battery powered clock chip connected via
```

Comment:
Please add a reference to a datasheet and link to a seller (like adafruit, sparkfun; not ebay or so because those links expire)

Reply:
Sure! I'll link to a datasheet. But do you really want a link to a specific seller? Basically you can get those chips everywhere - just look for DS1307 on Amazon, Aliexpress, ebay, whichever is your preferred dealer.

Reply:
Well, I don't care which seller it is specifically. It's just that I want links that don't go dead after a while (a mess to maintain)